core/src/block.rs

```diff
@@ -275,6 +275,7 @@ mod valid {
             block: SignedBlock,
```

Comment:
^ I suggest to add the new error condition into the doc above

`- Transaction in the genesis block is not signed by the genesis public key`